# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/mesa/hotfix-llvm-22-arm.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- mesa-26.0.2/src/gallium/auxiliary/gallivm/lp_bld_misc.cpp.vanilla	2026-03-13 18:52:17.555738317 +0100
+++ mesa-26.0.2/src/gallium/auxiliary/gallivm/lp_bld_misc.cpp	2026-03-13 18:55:11.810468818 +0100
@@ -331,7 +331,7 @@
       llvm::sys::getHostCPUFeatures(features);
    #endif
 
-   for (llvm::StringMapIterator<bool> f = features.begin();
+   for (llvm::StringMap<bool>::iterator f = features.begin();
         f != features.end();
         ++f) {
       MAttrs.push_back(((*f).second ? "+" : "-") + (*f).first().str());
